GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -239,14 +248,34 @@ boolean onKeyEventNative(int keyCode, int eventType) {
          * native side. It can be more than two, DOWN and UP, if the native
          * supports in the future.
          */
-        final int cKeyDown = 1;
 
-        int action = (eventType == cKeyDown) ? KeyEvent.ACTION_DOWN
-                : KeyEvent.ACTION_UP;
-        KeyEvent event = new KeyEvent(action, keyCode);
+        switch (eventType) {
```

Comment:
Thanks. It makes sense if the OVR library delivers all event types of these at least, not just the two of up and down.